examples/map/flatmap/Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: image
 image:
-	docker build -t "quay.io/numaio/numaflow-python/map-flatmap:v0.5.0" .
+	docker build -t "quay.io/numaio/numaflow-python/map-flatmap:v0.6.2" .
```

Comment:
Can we all use `0.6.1`? our sdks are now at 0.6.0 hence the next available minor version is 0.6.1